index.html

```diff
@@ -0,0 +1,238 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Portfolio</title>
+  <link rel="stylesheet" href="style.css">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+
+</head>
+
+<body>
+
+  <div class="header">
+
+    <div class="time">
+      <img class="icons image1" src="images/Time.png" alt="time">
+    </div>
+
+    <div class="connection">
+      <img class="icons image2" src="images/Cellular-Connection.png" alt="Cellular_Connection-path">
+      <img class="icons image3" src="images/Wifi.png" alt="wifi">
+      <img class="icons image4" src="images//Capacity.png" alt="Capacity">
+    </div>
+
+  </div>
+
+  <div class="name1">
+
+    <div class="nam1-text">
+      <h2 class="joe">Joe Doe</h2>
+    </div>
+
+    <div class="name1-img">
+      <img class="vector" src="images/Vector.png" alt="vector">
+      <img class="vector" src="images/Vector.png" alt="vector">
+      <img class="vector" src="images/Vector.png" alt="vector">
+    </div>
+  </div>
+
+
+  <div class="main">
+
+    <div class="name2">
+      <div class="laura laura1">
+        <img src="images/Ellipse18.svg" alt="elipse">
+      </div>
+
+      <div class="laura laura2">
+        <h1 class="laura-name"> Laura Jong</h1>
+
+        <div class="job">
+          <h2>Software Developper</h2>
+        </div>
+      </div>
+    </div>
+
+
+    <div class="description">
+      <p>I can help you build a product , feature or website Look through some of my work and experience! If you like
+        what you see and have a project you need coded, don’t hestiate to contact me.</p>
+    </div>
+
+    <div class="technologies">
+      <div class="languages">
+        <h5>Languages</h5>
+        <ul>
+          <li>Javascript</li>
+          <li>Ruby</li>
+          <li>Html</li>
+          <li>CSS</li>
+        </ul>
+      </div>
+      <div class="frameworks">
+        <h5>Frameworks</h5>
+        <ul>
+          <li>Bootstrap</li>
+          <li>Ruby on Rails</li>
+          <li>RSPec</li>
+          <li>CapyBara</li>
+          <li>Selenium</li>
+        </ul>
+      </div>
+
+      <div class="ellipse19">
+        <img src="images/Ellipse19.svg" alt="ellipse19">
+      </div>
```

Comment:
The same thing here as explained above please use this image as a `background-image` in the CSS file. Because this is also a decoration.